src/NServiceBus.Core/Timeout/Hosting/Windows/TimeoutPersisterReceiver.cs

```diff
@@ -19,6 +19,16 @@ class TimeoutPersisterReceiver : IDisposable
         public CriticalError CriticalError { get; set; }
         public Address DispatcherAddress { get; set; }
         public TimeSpan TimeToWaitBeforeTriggeringCriticalError { get; set; }
+        internal DateTime NextRetrieval { get; private set; } = DateTime.UtcNow;
```

Comment:
If being set early won't cause an issue then I'm all fine with it unless we vote that consistency in the implementation is desired.